packages/expect/src/jest-expect.ts

```diff
@@ -699,6 +699,42 @@ export const JestChaiExpect: ChaiPlugin = (chai, utils) => {
       )
     },
   )
+  def(
+    ['toHaveBeenCalledBefore'],
+    function (resultSpy: MockInstance) {
+      const expectSpy = getSpy(this)
+
+      const [firstExpectSpyCall] = expectSpy.mock.invocationCallOrder
```

Comment:
Why first? Shouldn't it be the smallest?

https://github.com/jest-community/jest-extended/blob/main/src/matchers/toHaveBeenCalledAfter.js#L43

Reply:
As the calls are ordered in the table, I thought that taking the first was enough, but maybe I'm wrong?